.github/workflows/generate-changelog.yml

```diff
@@ -31,3 +31,43 @@ jobs:
             git commit -m "$MSG"
             git push
           fi
+
+  slack-notification:
+    needs: [generate-changelog]
+    if: ${{ !cancelled() && needs.generate-changelog.result == 'failure' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send Slack message
```

Comment:
woundering if in the future it is worth having a more generic workflow for sending slack messages, not sure how much common code we have with other jobs sending messages.

Reply:
The structure is the same for all the instances where we send slack notifications, the payload is what changes. It's a potential improvement that we can look into it